test/elm/list/test.coffee

```diff
@@ -176,6 +176,9 @@ describe 'IndexOf', ->
     should(@nullList.exec(@ctx)).be.null
     should(@nullItem.exec(@ctx)).be.null
 
+  it 'should use equivalence to determine presence in List', ->
+    @listCode.exec(@ctx).should.equal 0
+
```

Comment:
What about the case where one of the two important fields (code or system) are not equal?

Reply:
Good idea--added tests for those cases.